frontend/src/Components/Customers/View.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Row } from "./Row";
+
+export function View(props) {
+  const [customers, setCustomers] = useState([]);
+  const [selectedRowIndex, setSelectedRowIndex] = useState(0);
+
+  useEffect(() => {
+    axios.get("/customers").then((cust) => {
+      setCustomers(cust.data);
+    });
+  }, []);
+
+  function onClick(index) {
+    setSelectedRowIndex(index);
+  }
+
+  function DisplayCustomers(searchText) {
+    if (searchText) {
+      customers = customers.filter(function (customer) {
```

Comment:
Aici cred ca reatribui valoarea din customers, fara sa updatezi si DOM-ul -- daca doresti sa editezi continutul din customers, ar trebui folosit setCustomers(customers.filter(...))

Reply:
Simt ca ar putea sa nu mai mearga, pls test inainte. 
Oricum, daca stii asta, vei evita f multe probleme legate de setState si unwanted side effects, been there:

 https://codeburst.io/explaining-value-vs-reference-in-javascript-647a975e12a0

Reply:
Nu a functionat codul. Am schimbat inapoi.